react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { activeIndex: 0 };
+    this.revolver = this.revolver.bind(this);
+    this.counter = this.counter.bind(this);
+    this.componentDidMount = this.componentDidMount.bind(this);
+    this.prevImg = this.prevImg.bind(this);
+  }
+
+  revolver() {
+    this.intervalId = setInterval(this.counter, 3000);
+  }
+
+  counter() {
```

Comment:
A better name for this method might have been `nextImg`, to fit better with the other method names